### ISSUE TITLE
Compatibility issue with haste | Too few arguments

### ISSUE THREAD

**Issue by fenepedia** [fenepedia/contao-klicktipp-gateway#9]
Folgender Fehler wird ins Log geschrieben:

```
[2024-08-13T08:23:57.323452+02:00] php.CRITICAL: Uncaught Error: Too few arguments to function Codefog\HasteBundle\StringParser::__construct(), 0 passed in /vendor/fenepedia/contao-klicktipp-gateway/src/Util.php on line 26 and exactly 2 expected {"exception":"[object] (ArgumentCountError(code: 0): Too few arguments to function Codefog\\HasteBundle\\StringParser::__construct(), 0 passed in /vendor/fenepedia/contao-klicktipp-gateway/src/Util.php on line 26 and exactly 2 expected at /vendor/codefog/contao-haste/src/StringParser.php:27)"} []

[2024-08-13T08:23:57.326708+02:00] request.CRITICAL: Uncaught PHP Exception ArgumentCountError: "Too few arguments to function Codefog\HasteBundle\StringParser::__construct(), 0 passed in/vendor/fenepedia/contao-klicktipp-gateway/src/Util.php on line 26 and exactly 2 expected" at /vendor/codefog/contao-haste/src/StringParser.php line 27 {"exception":"[object] (ArgumentCountError(code: 0): Too few arguments to function Codefog\\HasteBundle\\StringParser::__construct(), 0 passed in /vendor/fenepedia/contao-klicktipp-gateway/src/Util.php on line 26 and exactly 2 expected at /homepages/XY/vendor/codefog/contao-haste/src/StringParser.php:27)"} []
```

**Comment by fenepedia** [comment]
fixed with `2.0.2`

**Comment by fenepedia** [comment]
Error must also be corrected in 1.x. @fritzmg

**Comment by fritzmg** [comment]
> Error must also be corrected in 1.x. @fritzmg

Done and released in `1.2.2`.
